routes/routes.js

```diff
@@ -0,0 +1,29 @@
+require('dotenv').config()
+import { express, Twitter } from '../global_imports'
+import db from '../server/server'
+
+let router = express()
+let tweetStorage = []
+let client = new Twitter({
+  consumer_key: process.env.TWITTER_CONSUMER_KEY,
+  consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
+  access_token_key: process.env.TWITTER_ACCESS_TOKEN,
+  access_token_secret: process.env.TWITTER_ACCESS_SECRET
+})
+
+let getAll = () => db.getAllTweets()
+let getOne = () => db.getRandomTweet()
+let insert = (tweet) => { db.insertTweet(tweet) }
```

Comment:
seems nit-picky but removing the curly braces from lines 16 and 17 will keep the code looking consistent.